web-components/README.md

```diff
@@ -81,23 +81,15 @@ This section still requires discussions.<br />
 Main idea is to once the development is done, and ideally PR is already merged, the developer builds the artifacts and ships it to Milo as follows.<br />
 For non Milo consumers, we will support official releases to artifactory.
 
-# generate the build artifacts
-
-npm run build
-
-# copy the build artifacts to Milo's deps folder
-
+# Consumption in Milo
+run special build
 ```
-cp libs/*  ~/git/milo/libs/deps
+npm run build:milo
 ```
-
-# use copy / consume goals
-for now in web_components package only, you can use
+copy the build artifacts to Milo's deps folder:
 ```
-npm run copy
+cp milo_artifacts/*  ~/git/milo/libs/deps
```

Comment:
I'm thinking if milo as a git submodule to this repository would help to skip milo folder linking step?
```
$ git submodule add git@github.com:adobecom/milo.git milo
```
As submodule, milo is located in subfolder and should be separately committed to its repo when changes are made to its "libs" folder (+1 step to automation I guess).

Reply:
this sounds like a good discussion topic i think.. in this PR i change the folder location only, leaving consumption process as is for now